resources/completion.bash

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2021-present Fabien Potencier <fabien@symfony.com>
+#
+# This file is part of Symfony CLI project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Bash completions for the CLI binary
+#
+# References:
+#   - https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Console/Resources/completion.bash
+#   - https://github.com/posener/complete/blob/master/install/bash.go
+#   - https://github.com/scop/bash-completion/blob/master/completions/sudo
+#
+
+# this wrapper function allows us to let Symfony knows how to call the
+# `bin/console` using the Symfony CLI binary (to ensure the right env and PHP
+# versions are used)
+_{{ .App.HelpName }}_console() {
+  # shellcheck disable=SC2068
+  {{ .CurrentBinaryPath }} console $@
```

Comment:
One thing to discuss: here I used the absolute path to the binary.
Mainly to be sure it was using my local build during development. do you thing we should only use the binary name instead?
this require the user to have its PATH correctly setup though...

Reply:
Updated to automatically use the way the binary has been invoked to generate this script.